src/docfx/Errors.cs

```diff
@@ -404,7 +404,7 @@ public static Error UidConflict(string uid, IEnumerable<string> conflicts = null
             }
 
             var hint = conflicts.Count() > 5 ? "(Only 5 duplicates displayed)" : "";
-            return new Error(ErrorLevel.Error, "uid-conflict", $"Two or more documents have defined the same Uid '{uid}': {string.Join(',', conflicts.Take(5))}{hint}");
+            return new Error(ErrorLevel.Error, "uid-conflict", $"UID '{uid}' is defined in more than one file: {string.Join(',', conflicts.Take(5))}{hint}");
```

Comment:
consider replace all the `string.Join` in `Errors.cs` with `Join` method to ensure all values a quoted.

Reply:
Updated. Only `circular-reference` is using `string.Join()` in `Errors.cs` now